exchanges/order/order_types.go

```diff
@@ -133,6 +133,7 @@ type Detail struct {
 	ExecutedAmount       float64
 	RemainingAmount      float64
 	Cost                 float64
+	CostAsset            string
```

Comment:
Can we change this type from string to currency.Code? This is just for differentiation purposes.